specification/nhs-website-content-api.yaml

```diff
@@ -30,40 +30,24 @@ info:
       - Ratings and Reviews API
 
     ## API status and roadmap
-    This API has been moved from the nhs.uk developer API website. It is currently in Alpha phase. If you are an existing user, you need to sign up for the API in the new API Platform.
+    This API has moved from the nhs.uk developer API website. It is currently in development. If you are an existing user, you need to sign up for the API in the new API platform.
```

Comment:
This API has moved from the NHS website developer portal. It is currently in development. If you are an existing user, you need to create an account in the [Developer hub](https://digital.nhs.uk/developer).